.github/workflows/build.yml

```diff
@@ -35,10 +31,10 @@ jobs:
     steps:
       -
         name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
Don't think this is needed for this PR. I think we should use dependabot to update actions.